pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArithmeticFunctions.java

```diff
@@ -30,6 +30,16 @@ public class ArithmeticFunctions {
   private ArithmeticFunctions() {
   }
 
+  @ScalarFunction
+  public static double subtract(double a, double b) {
+    return a - b;
+  }
+
+  @ScalarFunction
+  public static double multiply(double a, double b) {
+    return a * b;
+  }
```

Comment:
We already have these here - https://github.com/apache/pinot/tree/4b26bf624965919c31a0fc6c6103ecab6531095b/pinot-common/src/main/java/org/apache/pinot/common/function/scalar/arithmetic.

If the goal is to also support additional aliases, we can add them here - https://github.com/apache/pinot/blob/4b26bf624965919c31a0fc6c6103ecab6531095b/pinot-query-planner/src/main/java/org/apache/pinot/calcite/sql/fun/PinotOperatorTable.java#L234-L236

Reply:
make sense, I can just add the alias

Reply:
done